app/views/labware/_asset_summary.html.erb

```diff
@@ -51,12 +51,37 @@
       <td><%= asset.human_barcode %></td>
     </tr>
   <% end %>
+  <% if asset.kind_of? TubeRack %>
+    <tr>
+      <th>Summary page</th>
+      <td><%= link_to "Summary for #{asset.human_barcode}",
+        tube_rack_summary_path(asset.human_barcode) %></td>
+    </tr>
+  <% end %>
   <% if  @source_plate&.human_barcode %>
     <tr>
       <th>Summary page</th>
       <td><%= link_to "Summary for #{@source_plate.human_barcode}", plate_summary_path(@source_plate.human_barcode) %></td>
     </tr>
   <% end %>
+  <% if asset.respond_to?(:tube_rack) && asset.tube_rack.present? %>
```

Comment:
Don't know if this is just me, but I would quite like a comment here explaining that this section is for Tubes that are in Tube Racks. Just because it's so generic, the view being for all Assets.

Reply:
Yes, is not clear at all. Adding a comment